app/views/sensors/add_sensor_reading.haml

```diff
@@ -0,0 +1,13 @@
+= button_tag "Add Sensor Reading manually", :id => 'unhide_sensor_reading_form', :class => "btn btn-primary"
```

Comment:
What's the matter with this possible duplication? Why do we get a `_partial` and a `template` with almost the same code?